File: Assignment 3/a3q3.java
import java.util.Scanner;
public class Prime
{
	public static void main(String [] args) 
	{
		int end = 10;
		
		/* for every number, all factors must be checked to be
		sure that it is prime.*/ //Check method below.***************
		
		
		System.out.println("enter a range of numbers to check as prime");
		
		int num1 = //scanner input
		int num 2 = //scanner input
		
		for(int i = num1; num
		
		
		
		
		
		
		
		for(int num = 1; num <= end; num++)
		{
			boolean isPrime =  true;
			
			for(int check = 2; check < num; check++)
			{
				if(num % check == 0)
					isPrime = false;
				System.out.println("\tChecking Divisor: " + check);
			}
			
				System.out.println(num + " " + isPrime);
		}
			
	
		
	}
	
	public static boolean isPrime(int in){
		boolean result = true;
		for(int i = 2; i<in; i++){
			if(in%i == 0)
				result = false;
		}
		return result;
		
	}

	
	
	
}

File: Assignment 3/cpp/a3q3.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main(){
    cout << "Enter an integer to print primes less than that integer: ";
    int num;
    cin >> num;
    bool prime;
    
    cout << "Primes: " << endl;

    for (int i = 3; i < num; i += 2) // odd numbers only
    {   
        prime = true; // Assume it is prime

        for (int j = 3; j < int(sqrt(i) + 1); j += 2) // odd numbers only
        {
            if (j != i && i % j == 0) // make sure we're not dealing with the same number
            {
                prime = false; 
            }
            
        }

        if (prime)
        {
            cout << i << endl;
        }
        
    }
    
}